src/Map.js

```diff
@@ -42,13 +42,13 @@ export function Map(props){
             
     		  wheelDeltaList.push(Math.abs(delta));
     		  var average = 0;
-    		  for(var i = 0; i< wheelDeltaList.length; i++){
+    		  for(let i = 0; i< wheelDeltaList.length; i++){  // frm: changed var to let to avoid duplicate def warning
```

Comment:
👍 

starting with es6, there's no reason to ever use `var` really

Reply:
I agree.